ServerCore/Pages/Puzzles/Edit.cshtml.cs

```diff
@@ -103,11 +103,28 @@ public async Task<IActionResult> OnPostAsync(int puzzleId)
 
             Puzzle.ID = puzzleId; // to be safe
 
+            string oldErrata = await (from Puzzle puzzle in _context.Puzzles
+                                      where puzzle.ID == puzzleId
+                                      select puzzle.Errata).FirstOrDefaultAsync();
+
             _context.Attach(Puzzle).State = EntityState.Modified;
 
             try
             {
                 await _context.SaveChangesAsync();
+
+                if (Puzzle.Errata != oldErrata)
```

Comment:
It looks like this would send mail if the errata was removed (with an empty string). I'd suggest checking String.IsNullOrWhitespace(Puzzle.Errata).

Reply:
Fair point.  We should probably still send mail in this case, just with a different message.